groupcache_test.go

```diff
@@ -285,6 +289,10 @@ func (p fakePeers) GetAll() []ProtoGetter {
 	return p
 }
 
+func (p fakePeers) GetURL(key string) string {
```

Comment:
You can remove this now; I think?

Reply:
hmm nop :D , fakePeers Implement the `ProtoGetter` with the new `GetURL()` method that need to be implemented

Reply:
`GetURL(key string) string` is a different interface than what `ProtoGetter` requires `GetURL() string `

Reply:
also, fakePeers is a collection of `[]ProtoGetter` not an implementation of the `ProtoGetter` interface, as such it doesn't have to implement the `ProtoGetter` interface, right?

Reply:
ah ok, missunderstand here :D , I removed it from `fakePeer``s` but added it on `fakePeer`